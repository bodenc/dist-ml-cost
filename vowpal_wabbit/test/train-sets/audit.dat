1 | feature
2 |namespace feature
3 | :1
4 |other_namespace :1
