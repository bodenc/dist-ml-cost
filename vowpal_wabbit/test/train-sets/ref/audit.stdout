0
	feature:118880:1:0@0
0
	namespace^feature:242309:1:0@0
0
	:0:1:0@0
0
	other_namespace^:207177:1:0@0
